Lib/idlelib/colorizer.py

```diff
@@ -69,17 +94,19 @@ def setdelegate(self, delegate):
             self.bind("<<toggle-auto-coloring>>", self.toggle_colorize_event)
             self.notify_range("1.0", "end")
         else:
-            # No delegate - stop any colorizing
+            # No delegate - stop any colorizing.
             self.stop_colorizing = True
             self.allow_colorizing = False
 
     def config_colors(self):
+        "Configure text widget tags with colors from tagdefs."
```

Comment:
Its a comment right?
```suggestion
        """Configure text widget tags with colors from tagdefs."""
```

Reply:
I prefer the single quotes, which leave space for 4 more chars,  for one line docstrings.